.github/README.md

```diff
@@ -1,16 +1,32 @@
-# UndertaleModTool
-*(seeing such an amazing tool...<br/>
-... fills you with DETERMINATION.)*
+<p align="center">
+  <img src="images/logo.png" alt="UndertaleModTool Logo"/>
+</p>
 
-Heya. I heard you like digging deep into Undertale data so I made a tool just for you! 
+![Discord](https://img.shields.io/discord/566861759210586112?label=Discord&logo=discord&logoColor=white) ![GitHub](https://img.shields.io/github/license/krzys-h/UndertaleModTool?logo=github)
 
-**Downloads** are **HERE**: :point_right::point_right::point_right: https://github.com/krzys-h/UndertaleModTool/releases :point_left::point_left::point_left:
+<p align="center">(seeing such an amazing tool it fills you with DETERMINATION.)</p>
+<p align="center">Heya. I heard you like digging deep into Undertale/Deltarune or any GameMaker game data so I made a tool just for you!</p>
+<p align="center">
+  <img src="images/flowey.gif" alt="Flowey"/>
+</p>
 
-**Have questions? Want to talk?** Join us on Discord! https://discord.gg/RxXpdwJ
+# Downloads
 
-![flowey](flowey.png)
+Looking for the most stable or the latest avaliable download? Both the latest stable release and the most cutting edge version are in the table below!
+
+| Releases 	| Status 	|
+|:---:	|----------	|
+| Stable[^1][^2] 	| ![GitHub release (latest by date)](https://img.shields.io/github/downloads/krzys-h/UndertaleModTool/latest/total) |
+| Bleeding edge[^1] 	| [![Build tool NET Bundled](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml/badge.svg)](https://github.com/krzys-h/UndertaleModTool/actions/workflows/build_net.yml) |
+
+[^1]: UndertaleModTool has multiple releases such has .net bundle, non single file, and single file the differenes on these releases are:
+      - .NET bundle: bundles the required .net framework version to run the tool
+      - Single file: just one exe an no extra dlls
+      - Non Single File: Just like above but it ships the dlls that are required by the tool
+[^2]: You can updated to the blending edge releases within the settings menu in UndertaleModTool.
```

Comment:
Those footnotes are shown alllll the way at the bottom of the file.
Would be nicer to actually display them right under the download table. 
Think that not as footnotes would help more for people to actually read the notes, as it would make them less missable.

Reply:
that's the way of how markdown handles footnotes, there's no much i can do about it

Reply:
There is a way, I'll make a new comment for this in a few to show potential solutions.